src/core/service.c

```diff
@@ -3338,6 +3372,11 @@ static bool service_may_gc(Unit *u) {
             control_pid_good(s) > 0)
                 return false;
 
+        /* Only allow collection of actually dead services, i.e. not those that are in the transitionary
+         * SERVICE_DEAD_RESTART/SERVICE_FAILED_RESTART states. */
```

Comment:
Typos

Reply:
which ones do you mean? i got the names of the service states slightly wrong. or something else?